### PR TITLE
Tell user they are in private channel when using active command

### PR DESCRIPTION
**Pull request by IncognitoJam**
Currently the bot will calculate the active users when the `~active` command is used in direct message - this doesn't need to happen because nobody else can be active here!

This commit makes the bot tell the user they are the only user active and then breaks, without performing any calculations.

**EDIT:** The bot will also use singular when there is only one active user.

### CODE REVIEW

File: src/discordtipbot/discordbot.cr [hunk 1]
```diff
@@ -381,18 +382,21 @@ class DiscordBot
...
         authors.each { |x| string = string + ", #{@cache.resolve_user(x).username}" }
       end
       string = string.lchop(", ")
-      reply(msg, "**#{msg.author.username}** rained a total of **#{amount} #{@config.coinname_short}** onto #{string}")
+      reply(msg, "**#{msg.author.username}** rained a total of **#{amount} #{@config.coinname_short}** (#{amount_each} #{@config.coinname_short} each) onto #{string}")
     end
   end
 
   def active(msg : Discord::Message)
+    return reply(msg, "**ERROR**: Only you are active here!") if private?(msg)
```
Comment (greenbigfrog) [inline src/discordtipbot/discordbot.cr]: This does look ok. Not sure if I'd prefer it to be a bit more clear about it being because of being in a DM.

Comment (z64) [reply src/discordtipbot/discordbot.cr]: This definitely can be improved.

Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: In what way?

Comment (z64) [reply src/discordtipbot/discordbot.cr]: The error is thrown because the channel is a private channel. The error does not tell them this.

Comment (greenbigfrog) [reply src/discordtipbot/discordbot.cr]: That's what I was trying to say

Comment (IncognitoJam) [reply src/discordtipbot/discordbot.cr]: I guess it's less of an error, and more about returning quickly without actually checking who is active. I could remove the 'error' part and just say 'only you are active'?

Comment (z64) [reply src/discordtipbot/discordbot.cr]: Just tell them why the error occurred.

"This cannot be done in a private / DM channel." or similar.

File: src/discordtipbot/discordbot.cr [hunk 2]
```diff
@@ -397,10 +397,14 @@ class DiscordBot
...
 
   def active(msg : Discord::Message)
+    return reply(msg, "You cannot use this command in a private channel!") if private?(msg)
+
     @bot.trigger_typing_indicator(msg.channel_id)
     authors = active_users(msg)
     return reply(msg, "No active users!") if authors.empty? || authors.nil?
-    reply(msg, "There are **#{authors.size}** active users ATM")
+
+    singular = authors.size == 0
```
Comment (greenbigfrog) [inline src/discordtipbot/discordbot.cr]: You want this to be `== 1`!

Comment (IncognitoJam) [reply src/discordtipbot/discordbot.cr]: Thanks, I've changed this now

**Pull request merged**